src/challenges/smileyFaces.js

```diff
@@ -0,0 +1,24 @@
+const countSmileys = (arr) => {
+  let count = 0
+  arr.forEach((smileyFace) => {
+    if (smileyFace.length === 2) {
+      if (
+        (smileyFace.indexOf(":") === 0 || smileyFace.indexOf(";") === 0) &&
+        (smileyFace.indexOf(")") === 1 || smileyFace.indexOf("D") === 1)
+      ) {
+        count += 1
+      }
+    } else if (smileyFace.length === 3) {
+      if (
+        (smileyFace.indexOf(":") === 0 || smileyFace.indexOf(";") === 0) &&
+        (smileyFace.indexOf("-") === 1 || smileyFace.indexOf("~") === 1) &&
+        (smileyFace.indexOf(")") === 2 || smileyFace.indexOf("D") === 2)
+      ) {
+        count += 1
+      }
+    }
```

Comment:
For practice can you change this to be a single condition?  The only 2 outcomes are the same.